coretex/entities/sample/network_sample.py

```diff
@@ -112,6 +119,24 @@ def _createSample(
 
         return cls.decode(response.json)
 
+    def modifiedSinceLastDownload(self) -> bool:
+        """
+            Checking if sample has been modified since last download, if the sample is already
+            stored locally
+
+            Returns
+            -------
+            bool -> False if sample has not changed since last download, True otherwise
+        """
+
+        if not self.zipPath.exists():
+            raise FileNotFoundError(f">> [Coretex] Sample file could not be found at {self.zipPath}. Cannot check if file has been modified since last download")
```

Comment:
Documentation of functions should contain which exceptions might be raised by the function.